code/modules/gear_presets/yautja.dm

```diff
@@ -45,6 +45,11 @@
 		translator_type = mob_client.prefs.predator_translator_type
 
 	new_human.equip_to_slot_or_del(new /obj/item/clothing/gloves/yautja/hunter(new_human, translator_type, caster_material, clan_rank), WEAR_HANDS)
+	if(new_human.client?.check_whitelist_status(WHITELIST_YAUTJA_COUNCIL))
+		new_human.equip_to_slot_or_del(new /obj/item/device/radio/headset/yautja/overseer(new_human), WEAR_L_EAR)
+	else
+		new_human.equip_to_slot_or_del(new /obj/item/device/radio/headset/yautja(new_human), WEAR_L_EAR)
```

Comment:
Dont think you need this else here, since everything initalises with a headset anyway, youre just overriding for COUNCIL

Reply:
OKAY!!!!

Reply:
didnt see you delte all the load gears so moot